frontend/src/styles/transactionDetail.ts

```diff
@@ -105,4 +105,8 @@ export const StyledTransactionDetails = styled(Box).attrs({
       margin-bottom: 4px;
     }
   }
+  
+  .time-out-message {
```

Comment:
Also, if you could remove this class and keep the same font as in the field description (same font-size and using the black color for text, and the blue color for links as shown below) - you probably don't need to style this component. 
<img width="1101" alt="Screenshot 2024-10-28 at 15 16 39" src="https://github.com/user-attachments/assets/86adef77-7839-4c8a-b3f0-45c3cb32c70c">

Reply:
Ideally, we should have a proper design for all the new features we implement, and these should be specified before implementing - we can try to do that going forward on one hand, and on the other hand, we have been provided with a modular design that we should follow to the best of our capacity when implementing new features. Here the text you're adding is similar in functionality to the clarification made near the date in the effective inclusion time, so we should keep the same style to be consistent throughout the page/website.